portfolio/src/main/webapp/script.js

```diff
@@ -31,12 +31,20 @@ function getMessage() {
 }
 
 
-function getMessageJSON() {
+function getCommentJSON() {
   fetch('/data')  // sends a request to /data
   .then(response => response.json()) // parses the response as JSON
-  .then((message) => { // now we can reference the fields in myObject!
-    console.log(message[0]);
-    console.log(message[1]);
-    console.log(message[2]);
+  .then((comments) => { // now we can reference the fields in myObject!
+    const commentsListEl = document.getElementById('data-container');
+    commentsListEl.innerHTML = ' ';
+    commentsListEl.appendChild(
+        createListElements('Comments: ' + comments.Comment)
```

Comment:
seems like it would be preferable to have a for loop here to unpack the comments array and have separate child elements for each comment? Otherwise, you'll see the funky JSON array formatting, which I doubt is what you want.

Reply:
I just changed my functions to follow the examples in the walkthrough more closely. Will that suffice?

Reply:
I'm not at all attached to a particular implementation. My primary goal is to make sure you understand what's going on. Thinking about design implications and usability are also important, which is why I made this particular comment.

Also, please think about review comments as a discussion about how to improve code, things to think about, etc., rather than a statement of "you must change this". I mean, some comments pretty much require fixing, but others are up to engineering judgment, and I'm more than happy to discuss (even debate) them with you :)